packages/e2e-test/src/commands/run.ts

```diff
@@ -427,24 +430,42 @@ async function testAppServe(pluginId: string, appDir: string) {
 }
 
 /** Drops PG databases */
-async function dropDB(database: string) {
-  const config = {
-    host: process.env.POSTGRES_HOST,
-    port: process.env.POSTGRES_PORT,
-    user: process.env.POSTGRES_USER,
-    password: process.env.POSTGRES_PASSWORD,
-  };
-
+async function dropDB(database: string, client: string) {
   try {
-    await pgtools.dropdb(config, database);
+    if (client === 'postgres') {
+      const config = {
+        host: process.env.POSTGRES_HOST ? process.env.POSTGRES_HOST : '',
+        port: process.env.POSTGRES_PORT ? process.env.POSTGRES_PORT : '',
+        user: process.env.POSTGRES_USER ? process.env.POSTGRES_USER : '',
+        password: process.env.POSTGRES_PASSWORD
+          ? process.env.POSTGRES_PASSWORD
+          : '',
```

Comment:
Did you add these empty-string fallbacks for some particular reason? Should not be needed right?

If they are to remain, you can just do `host: process.env.POSTGRES_HOST || ''` which is a bit less prone for mistakes

Reply:
Good call thanks!